kwanj-k/ride-my-way-API-V1
# #158603717 - Improved README

### What does this PR do?
Improves on the README 

### Description of Task to be completed
* Add clear instructions on installation
* Add clear instructions on testing on postman and pytest

### How should I test it?
Visit ``` https://github.com/kwanj-k/ride-my-way-API-V1/tree/ft-readme-%23158603717 ``` to see readme

### What PT stories?
[#158603717 ]https://www.pivotaltracker.com/story/show/158603717
